### ISSUE TITLE
Google account not logged in

### ISSUE THREAD

**Issue by j-liaman** [sebanc/brunch#1671]
lenovo g500 laptop
specification
CPU: Pentium 2020m
GPU: IGPU
RAM: 8GB DDR3
SSD 128GB
After selecting a personal account, I want to log in.
Please wait no further progress
What should I do

**Comment by ameyjoshi3068** [comment]
Have you download Chrome OS 103 with latest brunch?

**Comment by tejasraman** [comment]
@ameyjoshi3068 can you explain what this generic response is? Downgrade ≠ fix.

**Comment by tejasraman** [comment]
Hi @j-liaman could you please try octopus 103 just for my reference? This is obviously not a _you_ problem (see #1674) and I've asked the same to the people there.

Tejas

**Comment by ameyjoshi3068** [comment]
> 

To be honest I also think the same.

**Comment by ameyjoshi3068** [comment]
Now coming to problem,The exact same problem is also happening with me. in my case cpu is Pentium 4th gen. I tried many things on my own but can't solve it. the only way I got my mechine working is by downgrade so I thought it should be worth to mention this.

**Comment by tejasraman** [comment]
Hi @ameyjoshi3068 could you try Caroline or Octopus 105? I know that Caroline works with sandy bridge (with 105) and Haswell is newer so it should work... 

The problem is that Caroline only gets updates until 2024.....

**Comment by j-liaman** [comment]
I used Rammus up to version 100
I'm trying to install a new version 101, but there was a problem with the Google sign-in process. It doesn't work from version 101 or later.
Also, there was a problem while using version 100, but crosh was opened.
immediately shuts down

**Comment by ameyjoshi3068** [comment]
I tried caroline and octopus but none of them solved sign in issue.

**Comment by Nyta8591** [comment]
I also have Google login issue with this laptop https://chromebookdb.com/chromebook/hp-chromebook-14-14-ak040nr/ . Its EOL as of 2021 and latest OS was 93 i think. Anyone has 93version for Brunch and recovery before 100 i could try? 🙏

also audio and bluetooth doesnt work, and framework options werent able to fix audio. I managed to pull audio drives though from original OS. How can i mount them into Brunch plssss? 🙏🙏🙏

**Comment by tejasraman** [comment]
@Nyta8591 it has a baytrail Celeron, that won't work with 101 >

**Comment by Nyta8591** [comment]
> @Nyta8591 it has a baytrail Celeron, that won't work with >101. (Blame Google; they started to use ISAs that aren't supported on that CPU from >101)

Got it! Anyone has sub 100 version left? The oldest one of the website starts with 102…

also, will i be able to mount audio drives somehow plss? 🙏 could you please clarify what i should run in crosh?

**Comment by tejasraman** [comment]
Yeah, I have a version 99 image..

**Comment by Nyta8591** [comment]
> Yeah, I have a version 99 image... I'll send it once I go to my desktop

Thanks so much for all your help!

**Comment by tejasraman** [comment]
@Nyta8591 you're welcome!  You also need to enable this flag (#consumer-auto-update-toggle-allowed) to see the option in settings to stop updates

**Comment by Nyta8591** [comment]
> @Nyta8591 you're welcome! You also need to enable this flag (#consumer-auto-update-toggle-allowed) to see the option in settings to stop updates (otherwise your laptop will update past 100 and mess things up)

Awesome, will make sure to stop auto updates then. I really hope enabling audio flag for EOL chromebooks would be able to solve the issue, didnt work on the latest 111 though…

**Comment by tejasraman** [comment]
@Nyta8591 you need to enable the chromebook audio on brunch options

**Comment by tejasraman** [comment]
@Nyta8591 here is the image: [https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14526.69.0_rammus_recovery_stable-channel_mp-v2.bin.zip](https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14526.69.0_rammus_recovery_stable-channel_mp-v2.bin.zip) - it's an official rammus 100 image

**Comment by Nyta8591** [comment]
> @Nyta8591 here is the image: https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14526.69.0_rammus_recovery_stable-channel_mp-v2.bin.zip - it's an official rammus 100 image

many thanks for this, will try it out soon

**Comment by tejasraman** [comment]
You're welcome @Nyta8591

**Comment by Nyta8591** [comment]
do i need to exactly match the Brunch version, for example if I will try 94 Chrome OS, Brunch release should also be 94 or can be later?

**Comment by tejasraman** [comment]
I would recommend any Brunch version pre-103... 

This one should work: https://github.com/sebanc/brunch/releases/tag/r102-stable-20220625

**Comment by Nyta8591** [comment]
Not sure if I should try 94 rammus or 100.... given my Chromebook's last update was v93 I think, safer to go with 94?

**Comment by tejasraman** [comment]
No, go for 100... The AUE is arbitrary, they only dropped those ISAs later. 100 is a better release than 94 for a number of reasons

**Comment by Nyta8591** [comment]
got it, ok, will try it soon!

**Comment by Nyta8591** [comment]
> No, go for 100... The AUE is arbitrary, they only dropped those ISAs later. 100 is a better release than 94 for a number of reasons

Brunch 102+100 ChromeOS finally worked! I had to try different kernels to make audio work, and I only managed to get it working under Chromebook 4.4 kernel + “chromebook_audio” enabled under configurations. Bluetooth doesnt work though, but its minor… i guess it will be difficult to make it work?

**Comment by tejasraman** [comment]
Try iwlwifi backport???

**Comment by Nyta8591** [comment]
Hm, strangely i dont even see this option….
![Uploading IMG_4854.jpeg…]()

**Comment by tejasraman** [comment]
The photo is not loading... I'll check once I get back, I'm at Ikea right now

**Comment by Nyta8591** [comment]
![Flags](https://github.com/sebanc/brunch/assets/135348298/2d0687b1-3cae-4b4a-a86c-45012c0c2673)
sorry, here you go....

**Comment by tejasraman** [comment]
What kernel is that??

**Comment by Nyta8591** [comment]
I tried 5.1 with “chromebook_audio” enabled, but unfortunatelly it didnt fix the audio issue. Only the 4.4 chromebook kernel did

**Comment by Nyta8591** [comment]
Let me try activating native image, maybe it will fix the audio…

**Comment by tejasraman** [comment]
I said to enable `
native_chromebook_image
` too with 5.10

**Comment by tejasraman** [comment]
Sorry, you sent that.when I was typing

**Comment by Nyta8591** [comment]
Tried it, audio still not working, also tried on 5.4 kernel with same parameters you mentioned… thats a shame, i guess 4.4 chromebook is quite outdated (the only one that made my audio work, but only after debugging)

**Comment by tejasraman** [comment]
I'll look into this a little further... can you boot into a Linux USB, run "lshw" and tell me the audio chipset??

**Comment by Nyta8591** [comment]
If its not mentioned in any of the attached, I can do it tmrw only, since USB is in the room of someone who is asleep hahah 
[lspci, lsusb, lsmod.txt](https://github.com/sebanc/brunch/files/11644552/lspci.lsusb.lsmod.txt)
[byt-max98090.txt](https://github.com/sebanc/brunch/files/11644554/byt-max98090.txt)
[HiFi.txt](https://github.com/sebanc/brunch/files/11644555/HiFi.txt)

Many thanks for all your help! Wouldn't be able to make this machine work without you....

**Comment by tejasraman** [comment]
There's a line missing at the top of the file for the maxim98090 codec

Add `Syntax 2` to the top of the UCM2 file inside the ChromeOS shell (not crosh)

**Comment by Nyta8591** [comment]
Ok, will try tomorrow, going to sleep! Thanks so much once again

**Comment by tejasraman** [comment]
You're welcome @Nyta8591

**Comment by Nyta8591** [comment]
> There's a line missing at the top of the file for the maxim98090 codec
> 
> Add `Syntax 2` to the top of the UCM2 file inside the ChromeOS shell (not crosh)
> 
> And do this after **disabling native Chromebook image** (otherwise UCM2 won't apply)

Ok, tried googling, but still couldn’t figure out what you meant by CromeOS shell if its not Crosh… also not sure how to disable native chromebook image :(

**Comment by Nyta8591** [comment]
also, looks some games from Playstore don't load, like Roblox/Need for Speed, apparently thats a ChromeOs limitation on non-native hardware.... I think it loaded fine on Bliss OS (which runs on Android 12), but sadly audio doesn't work there either... Bliss admin said if I can provide the source code devs can add drivers to the next build... would you know by chance where to get the source code for audio drivers? 

Thanks so much!

**Comment by tejasraman** [comment]
> > There's a line missing at the top of the file for the maxim98090 codec
> 
> > 
> 
> > Add `Syntax 2` to the top of the UCM2 file inside the ChromeOS shell (not crosh)
> 
> > 
> 
> > And do this after **disabling native Chromebook image** (otherwise UCM2 won't apply)
> 
> 
> 
> Ok, tried googling, but still couldn’t figure out how to disable native chromebook image :( or add ‘Syntax 2’ to UCM2 file…

Type "shell" in crosh, and use nano (with sudo) to edit the UCM, and use alsaucm to load it

Go to brunch settings and redo without native chromebook image before doing this

**Comment by tejasraman** [comment]
> also, looks some games from Playstore don't load, like Roblox/Need for Speed, apparently thats a ChromeOs limitation on non-native hardware.... I think it loaded fine on Bliss OS (which runs on Android 12), but sadly audio doesn't work there either... Bliss admin said if I can provide the source code devs can add drivers to the next build... would you know by chance how to get the source code for audio drivers? 
> 
> 
> 
> Thanks so much!

When you say this, do they crash or not show up in the play store?

**Comment by Nyta8591** [comment]
> > also, looks some games from Playstore don't load, like Roblox/Need for Speed, apparently thats a ChromeOs limitation on non-native hardware.... I think it loaded fine on Bliss OS (which runs on Android 12), but sadly audio doesn't work there either... Bliss admin said if I can provide the source code devs can add drivers to the next build... would you know by chance how to get the source code for audio drivers?
> > Thanks so much!
> 
> When you say this, do they crash or not show up in the play store?

so they download, but when I try opening the apps, they crash....

**Comment by tejasraman** [comment]
Interesting… does this affect all apps?

**Comment by Nyta8591** [comment]
> Interesting… does this affect all apps?

no, just some apps.... but I was told that its a ChromeOS limitation on non-native hardware...

**Comment by tejasraman** [comment]
That shouldn’t be a thing… I’ll test the apps on a brunch system and tell you

**Comment by Nyta8591** [comment]
Just checked on Bliss OS (Android12), Roblox loads into starting menu, but then also fails.. btw, dmesg in Bliss gives that, do you know if its also alsa config files issue or smth else?

![Uploading IMG_4874.jpeg…]()

**Comment by Nyta8591** [comment]
> > > There's a line missing at the top of the file for the maxim98090 codec
> > 
> > 
> > > 
> > 
> > 
> > > Add `Syntax 2` to the top of the UCM2 file inside the ChromeOS shell (not crosh)
> > 
> > 
> > > 
> > 
> > 
> > > And do this after **disabling native Chromebook image** (otherwise UCM2 won't apply)
> > 
> > 
> > Ok, tried googling, but still couldn’t figure out how to disable native chromebook image :( or add ‘Syntax 2’ to UCM2 file…
> 
> Type "shell" in crosh, and use nano (with sudo) to edit the UCM, and use alsaucm to load it
> 
> Go to brunch settings and redo without native chromebook image before doing this

So i am using this guide to remount the original alsa configs i pulled from native ChromeOS, but I can't locate the correct alsa folder.... do you know how to locate it? Also, how to enter the developer mode, I am not sure if the system is rw to let me replace config files?  

https://askubuntu.com/questions/1189122/audio-chtmax98090-still-not-working-after-fix

![image](https://github.com/sebanc/brunch/assets/135348298/72d885ba-0db4-456c-8818-0c8ea5fa70ff)

**Comment by Nyta8591** [comment]
Also, I would like to sideload APKs and by following this guide https://beebom.com/how-sideload-android-apps-chromebook/ and sadly I don't see any "Linux (Beta)" Option...  

![image](https://github.com/sebanc/brunch/assets/135348298/ed7543a2-8ad6-4afe-9ff5-0fcc72061ec6)

**Comment by tejasraman** [comment]
Also, you'd need to ask MrChromebox if you can't use virtualization...

**Comment by tejasraman** [comment]
Tell him that you cant enable hardware assisted virtualization (on his GitHub issue tracker, that's more for these things)

**Comment by tejasraman** [comment]
> > > > There's a line missing at the top of the file for the maxim98090 codec
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > Add `Syntax 2` to the top of the UCM2 file inside the ChromeOS shell (not crosh)
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > And do this after **disabling native Chromebook image** (otherwise UCM2 won't apply)
> > > 
> > > 
> > > Ok, tried googling, but still couldn’t figure out how to disable native chromebook image :( or add ‘Syntax 2’ to UCM2 file…
> > 
> > Type "shell" in crosh, and use nano (with sudo) to edit the UCM, and use alsaucm to load it
> > 
> > Go to brunch settings and redo without native chromebook image before doing this
> 
> So i am using this guide to remount the original alsa configs i pulled from native ChromeOS, but I can't locate the correct alsa folder.... do you know how to locate it? Also, how to enter the developer mode, I am not sure if the system is rw to let me replace config files?  
> 
> https://askubuntu.com/questions/1189122/audio-chtmax98090-still-not-working-after-fix
> 
> ![image](https://github.com/sebanc/brunch/assets/135348298/72d885ba-0db4-456c-8818-0c8ea5fa70ff)
> 
> 
> 

Use "nano /usr/share/alsa/ucm2/name_of_file.cfg" in shell to edit it.

**Comment by Nyta8591** [comment]
> > > > > There's a line missing at the top of the file for the maxim98090 codec
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > Add `Syntax 2` to the top of the UCM2 file inside the ChromeOS shell (not crosh)
> > > > 
> > > > 
> > > > > 
> > > > 
> > > > 
> > > > > And do this after **disabling native Chromebook image** (otherwise UCM2 won't apply)
> > > > 
> > > > 
> > > > Ok, tried googling, but still couldn’t figure out how to disable native chromebook image :( or add ‘Syntax 2’ to UCM2 file…
> > > 
> > > 
> > > Type "shell" in crosh, and use nano (with sudo) to edit the UCM, and use alsaucm to load it
> > > Go to brunch settings and redo without native chromebook image before doing this
> > 
> > 
> > So i am using this guide to remount the original alsa configs i pulled from native ChromeOS, but I can't locate the correct alsa folder.... do you know how to locate it? Also, how to enter the developer mode, I am not sure if the system is rw to let me replace config files?
> > https://askubuntu.com/questions/1189122/audio-chtmax98090-still-not-working-after-fix
> > ![image](https://user-images.githubusercontent.com/135348298/243361714-72d885ba-0db4-456c-8818-0c8ea5fa70ff.png)
> 
> Use "nano /usr/share/alsa/ucm2/name_of_file.cfg" in shell to edit it.

ucm2 directory doesn't exist and ucm gives me an empty file? 

![image](https://github.com/sebanc/brunch/assets/135348298/e4cda97f-ee25-40ed-9834-02279862396f)
![image](https://github.com/sebanc/brunch/assets/135348298/dbf46d14-fb2e-4bf1-9348-f0725d34add0)

**Comment by tejasraman** [comment]
It doesn't exist if native_chromebook_image is on. Turn it off and try after that.

**Comment by tejasraman** [comment]
Check inside ucm2 and use sudo

**Comment by Nyta8591** [comment]
> Check inside ucm2 and use sudo

There is nothing under ucm2, but with sudo i managed to alrer this one /usr/share/alsa/ucm/byt-max98090/byt-max98090.conf

do i enable native chromebook image now via  sudo edit-brunch-conf to check if it worked?

**Comment by tejasraman** [comment]
Ok, something's wrong..... I need to check with these versions myself. I'll install chrome100 on a legacy laptop and tell you what to do to get audio

**Comment by Nyta8591** [comment]
> Ok, something's wrong..... I need to check with these versions myself. I'll install chrome100 on a legacy laptop and tell you what to do to get audio

thanks so much for all your help, you are awesome!!!

**Comment by Nyta8591** [comment]
> Also, you'd need to ask MrChromebox if you can't use virtualization...

https://github.com/MrChromebox/firmware/issues/474#issuecomment-1577008553

he replied....

**Comment by tejasraman** [comment]
Also, you can sideload an apk by double clocking on it because Brunch comes with dev mode on be default.

**Comment by Nyta8591** [comment]
I get this :( 

![image](https://github.com/sebanc/brunch/assets/135348298/c03891ee-c4fb-4de9-8641-c37c3d34df89)
![image](https://github.com/sebanc/brunch/assets/135348298/e2fc7067-ea92-4adc-9763-fdac280b3498)

**Comment by tejasraman** [comment]
Ugh, I just remembered: The Chromebook variants of Baytrail never had VTX (Haswell and Braswell did)

See [https://chromium.googlesource.com/chromiumos/docs/+/master/containers_and_vms.md#baytrail-systems](this)

**Comment by tejasraman** [comment]
Are you using a work account on the Chromebook? Is unknown sources enabled in arc++/Android settings?

**Comment by Nyta8591** [comment]
> Are you using a work account on the Chromebook? Is unknown sources enabled in arc++/Android settings?

this is my nephew's, so its child's account (managed by Family Link), but I don't think it should affect it? 

![image](https://github.com/sebanc/brunch/assets/135348298/e9a14da8-5c6e-49b4-9575-50b6cca28dd8)
![image](https://github.com/sebanc/brunch/assets/135348298/c67ffc8b-66fe-4e20-8d32-d9322282f57e)

**Comment by tejasraman** [comment]
You need to go to family link settings and enable it under device settings.

**Comment by tejasraman** [comment]
![PXL_20230605_170259709.jpg](https://github.com/sebanc/brunch/assets/96961804/e7a3e40a-f208-4aea-945f-4c5bf4e82621)

**Comment by tejasraman** [comment]
You need to use the cb4.4 kernel because that ucm is ucm1 only. You could try the latest brunch version - it has a cb5.10 that will work.....

**Comment by Nyta8591** [comment]
so, i can try creating an image of v100 ChromeOS rammus + r113 Brunch OS (https://github.com/sebanc/brunch/releases/tag/r113-stable-20230603) ?

**Comment by Nyta8591** [comment]
> You need to go to family link settings and enable it under device settings.

Very iteresting... its impossible to sideload apps on an account which is under 13... I guess I will be creating a new account hahah
![image](https://github.com/sebanc/brunch/assets/135348298/79afe54d-79f5-49b3-9867-eeff12a508cd)

**Comment by Nyta8591** [comment]
> so, i can try creating an image of v100 ChromeOS rammus (https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14526.69.0_rammus_recovery_stable-channel_mp-v2.bin.zip) + r113 Brunch OS (https://github.com/sebanc/brunch/releases/tag/r113-stable-20230603) ?

and for this, did i understand this correctly?

**Comment by tejasraman** [comment]
> > so, i can try creating an image of v100 ChromeOS rammus (https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14526.69.0_rammus_recovery_stable-channel_mp-v2.bin.zip) + r113 Brunch OS (https://github.com/sebanc/brunch/releases/tag/r113-stable-20230603) ?
> 
> and for this, did i understand this correctly? 

Maybe... Try it and tell me please :)

**Comment by Nyta8591** [comment]
> > > so, i can try creating an image of v100 ChromeOS rammus (https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14526.69.0_rammus_recovery_stable-channel_mp-v2.bin.zip) + r113 Brunch OS (https://github.com/sebanc/brunch/releases/tag/r113-stable-20230603) ?
> > 
> > 
> > and for this, did i understand this correctly?
> 
> Maybe... Try it and tell me please :)

Ok, Rammus 100+Brunch 113 just wouldn't load, stuck at the starting screen "RootFS is being rebuilt".... maybe i can try 103 rammus + 113 Brunch?

**Comment by Nyta8591** [comment]
> No, you need.to do it with Linux again.....

you mean installing it via USB with Ubuntu? I can try.... pls let me know what I should run to update GLIBC...

**Comment by tejasraman** [comment]
The glibc in chrome100 is too old to update that far. Using Linux to reinstall it would bypass that.

**Comment by Nyta8591** [comment]
> The glibc in chrome100 is too old to update that far. Using Linux to reinstall it would bypass that.

so how do I update GLIBC via Ubuntu USB? I ran these commands and it says "libc6 is already the newest version (2.37)", but I still get the same error in ChromeOS.... so USB commands were either not saved or not recognized by ChromeOS? 

"sudo apt update
sudo apt install libc6 "

![image](https://github.com/sebanc/brunch/assets/135348298/8a390efc-874b-4a47-ba0b-83242b17230f)

**Comment by Nyta8591** [comment]
> No, you need to run `chromeos_install.sh` inside Ubuntu to reinstall, but use the version 113 brunch to test

so, rammus 100+Brunch 113:

5.10 kernel (with enabled "chromebook audio" + native chromeos image) --> no sound
chromebook 5.10 kernel (with/without enabled "chromebook audio" ) --> no sound
chromebook 4.19 kernel (with/without enabled "chromebook audio" ) --> no sound

Strange?

**Comment by Nyta8591** [comment]
> 

thats the first option I tried... still no soud :(

**Comment by Nyta8591** [comment]
> You need to use the cb4.4 kernel because that ucm is ucm1 only. You could try the latest brunch version - it has a cb5.10 that will work... The cryptohome changes in 103 make me doubtful but maybe it will work??

maybe as last test, i could try rammus 103 + brunch 113? Do you have rammus 103 image left by chance?

**Comment by tejasraman** [comment]
> > You need to use the cb4.4 kernel because that ucm is ucm1 only. You could try the latest brunch version - it has a cb5.10 that will work... The cryptohome changes in 103 make me doubtful but maybe it will work??
> 
> maybe as last test, i could try rammus 103 + brunch 113? Do you have rammus 103 image left by chance?

103 won’t work with Baytrail, it will crash.

 I have the link to pretty much every single image to date :) If you want to try, here: [https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14816.82.0_rammus_recovery_stable-channel_mp-v2.bin.zip](https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14816.82.0_rammus_recovery_stable-channel_mp-v2.bin.zip)

**Comment by Nyta8591** [comment]
> > > You need to use the cb4.4 kernel because that ucm is ucm1 only. You could try the latest brunch version - it has a cb5.10 that will work... The cryptohome changes in 103 make me doubtful but maybe it will work??
> > 
> > 
> > maybe as last test, i could try rammus 103 + brunch 113? Do you have rammus 103 image left by chance?
> 
> 103 won’t work with Baytrail, it will crash.
> 
> I have the link to pretty much every single image to date :) If you want to try, here: https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14816.82.0_rammus_recovery_stable-channel_mp-v2.bin.zip

got it! ok, then I guess my only option is 4.4 kernel with rammus 100+brunch103 :) not sure why this one doesn't want to work ( 5.10-chromebook WITH native-chromebook-image AND chromebook-audio)....

**Comment by tejasraman** [comment]
> > > > You need to use the cb4.4 kernel because that ucm is ucm1 only. You could try the latest brunch version - it has a cb5.10 that will work... The cryptohome changes in 103 make me doubtful but maybe it will work??
> > > 
> > > 
> > > maybe as last test, i could try rammus 103 + brunch 113? Do you have rammus 103 image left by chance?
> > 
> > 103 won’t work with Baytrail, it will crash.
> > 
> > I have the link to pretty much every single image to date :) If you want to try, here: https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14816.82.0_rammus_recovery_stable-channel_mp-v2.bin.zip
> 
> got it! ok, then I guess my only option is 4.4 kernel with rammus 100+brunch103 :) not sure why this one doesn't want to work ( 5.10-chromebook WITH native-chromebook-image AND chromebook-audio).... 

brunch 102 right?? The link to chrome 100 was in one of my replies

You could try fydeos as well.

**Comment by Nyta8591** [comment]
> > > > > You need to use the cb4.4 kernel because that ucm is ucm1 only. You could try the latest brunch version - it has a cb5.10 that will work... The cryptohome changes in 103 make me doubtful but maybe it will work??
> > > > 
> > > > 
> > > > maybe as last test, i could try rammus 103 + brunch 113? Do you have rammus 103 image left by chance?
> > > 
> > > 
> > > 103 won’t work with Baytrail, it will crash.
> > > I have the link to pretty much every single image to date :) If you want to try, here: https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14816.82.0_rammus_recovery_stable-channel_mp-v2.bin.zip
> > 
> > 
> > got it! ok, then I guess my only option is 4.4 kernel with rammus 100+brunch103 :) not sure why this one doesn't want to work ( 5.10-chromebook WITH native-chromebook-image AND chromebook-audio)....
> 
> brunch 102 right?? The link to chrome 100 was in one of my replies
> 
> You could try fydeos as well.

yeah, 102 sorry. Fyde OS i already tried, I couldn't enable Google services there for some reason and Brunch seems much smoother even with 4.4 kernel!

**Comment by Nyta8591** [comment]
Do you know which version of FydeOS or OpenFyde i should install for my Chromebook? I am afraid I will run into the same problems with the newest versions as with Brunch....

**Comment by tejasraman** [comment]
Try this one: https://fydeos.io/download/pc/intel-hd

**Comment by tejasraman** [comment]
> I couldnt activate Gapps on this one, i belive could be the same issue with latest versions being incompatible with that CPU? 
> 
> > Try this one: https://fydeos.io/download/pc/intel-hd
> 
> 

Sorry for being late, my email put your message into spam. I'll test on another laptop, but the lack of vtd might explain it
